Tests.Integration.Presentation.Web/Organizations/StsOrganizationSynchronizationApiTest.cs

```diff
@@ -546,6 +546,146 @@ public async Task Can_PUT_UPDATE_Consequences_With_Conversion_Consequences()
+            var expectedConversionUuid = Guid.NewGuid();
+            DatabaseAccess.MutateEntitySet<OrganizationUnit>(repo =>
+            {
+                expectedConversionUuid = repo
+                    .AsQueryable()
+                    .Where(x => x.Organization.Uuid == targetOrgUuid &&
+                                x.Origin == OrganizationUnitOrigin.STS_Organisation && !x.Children.Any())
+                    .ToList()
+                    .RandomItem()
+                    .Uuid;
+            });
+            
+            //Make sure it is in use so it will not be deleted, but converted
+            await ItContractV2Helper.PostContractAsync(globalAdminToken.Token,
+                new CreateNewContractRequestDTO()
+                {
+                    Name = A<string>(),
+                    OrganizationUuid = targetOrgUuid,
+                    Responsible = new() { OrganizationUnitUuid = expectedConversionUuid }
+                });
+
+            using var otherConsequencesResponse = await SendGetUpdateConsequencesAsync(targetOrgUuid, firstRequestLevels, cookie);
+            Assert.Equal(HttpStatusCode.OK, otherConsequencesResponse.StatusCode);
+            var otherConsequencesBody = await otherConsequencesResponse.ReadResponseBodyAsKitosApiResponseAsync<ConnectionUpdateConsequencesResponseDTO>();
+            var otherConsequences = otherConsequencesBody.Consequences.ToList();
+
+            //Log deletion, renaming and conversion changes
+            using var putResponse = await SendPutUpdateConsequencesAsync(targetOrgUuid, firstRequestLevels, cookie);
+            Assert.Equal(HttpStatusCode.OK, putResponse.StatusCode);
+
+            //Relocation consequences
+            var uuidOfExpectedMoval = Guid.NewGuid();
+            DatabaseAccess.MutateEntitySet<OrganizationUnit>(repo =>
+            {
+                var twoLeafs = repo
+                    .AsQueryable()
+                    .Where(x => x.Organization.Uuid == targetOrgUuid && x.Origin == OrganizationUnitOrigin.STS_Organisation && !x.Children.Any())
+                    .ToList()
+                    .RandomItems(2)
+                    .ToList();
+
+                var firstLeaf = twoLeafs.First();
+                var secondLeaf = twoLeafs.Last();
+
+                //Make first leaf parent of second leaf
+                secondLeaf.ParentId = firstLeaf.Id;
+
+                //Add a native units so we can check that it is moved along with the moved unit
+                var organization = secondLeaf.Organization;
+                organization.AddOrganizationUnit(new OrganizationUnit { Organization = organization, Name = "Native test unit", Uuid = uuidOfExpectedMoval, ObjectOwner = secondLeaf.ObjectOwner, LastChangedByUser = secondLeaf.LastChangedByUser }, secondLeaf);
+            });
+
+            using var relocationConsequencesResponse = await SendGetUpdateConsequencesAsync(targetOrgUuid, firstRequestLevels, cookie);
```

Comment:
Ok so let's take this again... the point of consolidating this method into one was not to just copy all of the tests into one.. it was to save slow, external calls so let's me be more precise:
- Make a setup do the amount of slow imports that you need. I see 3 update calls, but in order to deal with a collection "more than one" should be ok. See if you can squeeze the test down to two updates, thereby saving 
- A call to GetConsequences
- A call to PUT

That at least saves us some time there